docs/start/introduction.md

```diff
@@ -0,0 +1,84 @@
+# A Friendly Introduction
+
+## Why use flowchem?
+
+Flowchem was designed to assist in the development of automation in laboratories. Many electronic devices used in 
+laboratories have different providers and different types of communication. To automate, for example, a set of devices 
+to operate a process automatically, it is necessary:
+
+* Define a programming coding language;
+
+* Search with providers for ways to access the device using the chosen language;
+
+* Develop the communication system using computer signal inputs like serial and Ethernet;
+
+* Integrate all devices into a single system, where it is possible to start building protocols and instructions.
+
+In addition to all the complexity involved in the steps presented, a factor that can be stressful is if there is a need 
+to change a device during the construction or operation of the platform. This change will require rework to build the 
+new device system, especially if it comes from a different supplier (with other communication, functions, and methods).
+
+Flowchem connects with each device, ensuring easy access to all available commands through a web interface. This means 
+that the user will not have to limit themselves to using a specific program language, as the connection to the devices 
+can be easily accessed with a web data request package, which is present in most program languages (i.e., python, java, 
+c++, HTML, and others).
+
+Flowchem also has a high level of abstraction concerning devices. This abstraction occurs through the use of class, 
```

Comment:
if it is friendly introduction, we might need to explain abstraction. and Concerning is not common to used. class might also be something we need to explain.

Reply:
I do not think so. Thats code basics anyone can look up anywhere in no time. And abstraction is a general concept - how abstraction is used in flowchem is explained in the following